runtime/oti/vm_api.h

```diff
@@ -4090,15 +4090,25 @@ typedef enum {
 void*
 J9_GetInterface(J9_INTERFACE_SELECTOR interfaceSelector, J9PortLibrary* portLib, void *userData);
 
-/* -------------------- BytecodeInterpreter.cpp ------------ */
+/* -------------------- BytecodeInterpreterFull.cpp ------------ */
 
 /**
-* @brief Execute the bytecode loop
+* @brief Execute the bytecode loop (full pointers)
+* @param currentThread
+* @return UDATA the action to take upon return to the builder interpreter
+*/
+UDATA
+bytecodeLoopFull(J9VMThread *currentThread);
+
+/* -------------------- BytecodeInterpreterCompressed.cpp ------------ */
+
+/**
+* @brief Execute the bytecode loop (compressed pointers)
 * @param currentThread
 * @return UDATA the action to take upon return to the builder interpreter
 */
 UDATA  
-bytecodeLoop(J9VMThread *currentThread);
+bytecodeLoopCompressed(J9VMThread *currentThread);
```

Comment:
No need to ifdef prototypes, though I'd be tempted to move these to the place that uses them, since they're so specific (and no one should be calling them directly, ever).

Reply:
These are only used in jvminit.c. The Full/Compressed prototypes could be moved to that file?

Reply:
I think that would be ideal - it means no one could accidentally call them. I suppose for complete safety, the prototypes should be ifdeffed in jvminit.c.

Yes, please move the debug version(s) to jvminit.c as well.